packages/mui-codemod/src/deprecations/mobile-stepper-props/test-cases/expected.js

```diff
@@ -0,0 +1,19 @@
+import MobileStepper from '@mui/material/MobileStepper';
+import { MobileStepper as MyMobileStepper } from '@mui/material';
+
+<MobileStepper variant="progress" slotProps={{
+  progress: { variant: 'determinate' }
+}} />;
+<MyMobileStepper
+  variant="progress"
+  slotProps={{
+    progress: {
+      ...{ variant: 'determinate' },
+
+      ...{
+        sx: { width: '100%' },
+      }
+    },
+  }} />;
+
+<CustomMobileStepper LinearProgressProps={{ variant: 'determinate' }} />;
```

Comment:
```suggestion
<CustomMobileStepper slotProps={{progress:{varaint:"dterminate"}}} />;
```
This transformation gt missed

Reply:
This is correct. The codemod should not transform CustomMobileStepper that's not imported from Material UI